docs/index.md

```diff
@@ -46,18 +47,18 @@ Enqueue is an MIT-licensed open source project with its ongoing development made
     - [Run unique job](job_queue/run_unique_job.md)
     - [Run sub job(s)](job_queue/run_sub_job.md)
 * [EnqueueBundle (Symfony)](#enqueue-bundle-symfony).
-    - [Quick tour](bundle/quick_tour.md)
-    - [Config reference](bundle/config_reference.md)
-    - [Cli commands](bundle/cli_commands.md)
-    - [Message producer](bundle/message_producer.md)
-    - [Message processor](bundle/message_processor.md)
-    - [Async events](bundle/async_events.md)
-    - [Async commands](bundle/async_commands.md)
-    - [Job queue](bundle/job_queue.md)
-    - [Consumption extension](bundle/consumption_extension.md)
-    - [Production settings](bundle/production_settings.md)
-    - [Debugging](bundle/debugging.md)
-    - [Functional testing](bundle/functional_testing.md)
```

Comment:
This is a BC break. There are links to those pages on the Internet. I don't mind moving them to symfony namespace. IMO `symfony` section is a better one. but to do so, we should add pages that would guide people coming from outside to right place

Reply:
I've reverted this change for now. On my local build I had issues with "bundle" as directory name - it seemed that navigation markup is not read from this directory.